File: 30-Days-Of-Code/Day 5/Loops.java
import java.io.*;
import java.util.*;     //Not Required
import java.text.*;     //Not Required
import java.math.*;     //Not Required
import java.util.regex.*;   //Not Required

public class Solution {

    public static void main(String[] args) {
        Scanner in = new Scanner(System.in);
        int n = in.nextInt();   //taking input in 'n'
        int i;
    
        //here the loop is.
        for(i=1;i<=10;i++)
        System.out.println(n + " x " + i + " = " + n*i);
    }
}

File: 30-Days-Of-Code/Day 5/Loops.c
#include <math.h>   //Not Required  
#include <stdio.h>  
#include <string.h> //Not Required
#include <stdlib.h> //Not Required
#include <assert.h> //Not Required
#include <limits.h> //Not Required
#include <stdbool.h>    //Not Required

int main() {
    int n, i;
    scanf("%d", &n);
    
    //here the loop is.
    for(i=1;i<=10;i++)
    printf("%d x %d = %d\n", n, i, n*i);
        
    return 0;
}
